### PR TITLE
[37] unit test for login api endpoint

### PR DESCRIPTION
**Pull request by stanislawK**
To run tests - get into the running backend container and type pytest

### CODE REVIEW

File: backend/tests/conftest.py [hunk 1]
```diff
@@ -28,3 +29,55 @@ def app():
...
+
+@pytest.fixture
+def client(app):
+    """A test client for the app."""
+    return app.test_client()
+
+
+@pytest.fixture
+def _db():
+    """Create and configure a new db instance for pytest-flask-sqlalchemy"""
```
Comment (w1stler) [inline backend/tests/conftest.py]: Period at the end.

File: backend/tests/test_login.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import pytest
+
+
+def test_login_with_valid_user(client, new_user, registered_user):
+    """
+    GIVEN registerd user and app instance
+    WHEN login with valid user data
+    THEN check if access token is in response
```
Comment (w1stler) [inline backend/tests/test_login.py]: As in the previous PR.

File: backend/tests/conftest.py [hunk 3]
```diff
@@ -28,3 +29,55 @@ def app():
...
+        yield db
+
+    os.close(db_fd)
+    os.unlink(db_path)
+
+
+@pytest.fixture
+def new_user():
+    user = {'username': 'TestName',
+            'password': 'TestPassword'}
```
Comment (w1stler) [inline backend/tests/conftest.py]: I don't like this indentation.

File: backend/tests/test_login.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                                     'password': new_user['password']
+                                    })
+    response = rv.get_json()
+    assert rv.status == '201 CREATED'
+    assert response['access_token']
+
+
+def test_login_with_invalid_password(client, new_user, registered_user):
+    """
+    GIVEN registred user and app instance
```
Comment (w1stler) [inline backend/tests/test_login.py]: typo (registered)

**Comment by OtisRed** [comment]
@kristhina review, please (make start, make bash, make test)

**Pull request merged**